src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -204,10 +218,7 @@
         {:id resource-dropdown-id
          :items (->> resources (filter :enabled) (remove :archived))
          :item-key :id
-         :item-label #(str (:resid %)
-                           " (org: "
-                           (get-in % [:organization :organization/short-name language])
-                           ")")
+         :item-label #(resource-label % language counts)
```

Comment:
Minor thing here, I would prefer to send the count of the one resource only here, not the counts of all resources to the `resource-label` function. The responsibilities are a bit mixed. I'm not insisting on changing this.

Reply:
I am not sure what implications it might have, maybe I can leave it like this for now?